readme.md

```diff
@@ -0,0 +1,57 @@
+# Hello-microverse
+
+This project is a part of the microverse curriculum.
+<br>Objectif: Learn how to use GitHub in a professional way
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML & CSS
+- VsCode
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] [OPTIONAL] I would suggest to delete all the heading you are not using, otherwise it may  be confusing 😉